.ci_support/linux_ppc64le_python3.8.____cpython.yaml

```diff
@@ -22,3 +20,6 @@ python:
 - 3.8.* *_cpython
 target_platform:
 - linux-ppc64le
+zip_keys:
```

Comment:
Not sure where this comes from. Maybe auto-generated?

Reply:
Yea this is autogenerated and can be ignored